bootstrap.yml

```diff
@@ -2768,6 +2822,59 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: findutils
+    default: false
+    source:
+      subdir: ports
+      git: 'https://git.savannah.gnu.org/git/findutils.git'
+      tag: 'v4.7.0'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.11
+        - host-libtool
+        - host-pkg-config
+      regenerate:
+        - args: ['./bootstrap']
+        - args: ['cp',
+            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
+            '@THIS_SOURCE_DIR@/build-aux/']
+    tools_required:
+      - system-gcc
+      - host-python
+    configure:
+      - args:
+        - '@THIS_SOURCE_DIR@/configure'
+        - '--host=x86_64-managarm'
+        - '--prefix=/usr'
+        - '--without-selinux'
+        - '--disable-nls'
+        - '--localstatedir=/var/lib/locate'
+        environ:  
+          'PYTHON': '@BUILD_ROOT@/tools/host-python/bin/python3'
+    build:
+      - args: ['make', '-j@PARALLELISM@']
+      - args: ['make', 'install']
+        environ:
+          DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: bzip2
+    default: false
+    source:
+      subdir: ports
+      url: 'https://www.sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz'
+      format: 'tar.gz'
+      extract_path: 'bzip2-1.0.8'
+    tools_required:
+      - system-gcc
+    configure:
+      - args: ['cp', '-r', '@THIS_SOURCE_DIR@/.', '@THIS_BUILD_DIR@']
+      - args: 'sed -i s/"all: libbz2.a bzip2 bzip2recover test"/"all: libbz2.a bzip2 bzip2recover"/ @THIS_BUILD_DIR@/Makefile'
+    build:
+      - args: ['make', 'CC=x86_64-managarm-gcc', 'CFLAGS=-fPIC', '-f', 'Makefile-libbz2_so']
+      - args: ['make', 'clean']
+      - args: ['make', 'CC=x86_64-managarm-gcc', 'CFLAGS=-fPIC', '-j@PARALLELISM@']
```

Comment:
Why do we run `make` -> `make clean` -> `make` again?

Reply:
we run make on the makefile called Makefile-libbz2_so, so everything builds against a shared libbz2, this is in line with the LFS instructions found [here](http://www.linuxfromscratch.org/lfs/view/systemd/chapter06/bzip2.html)